ntoskrnl/cc/copy.c

```diff
@@ -294,6 +294,8 @@ CcPerformReadAhead(
 
 /*
  * @unimplemented
+ *
+ * 'Retrying' is actually used as a CC_CAN_WRITE_RETRY.
```

Comment:
This was added on 945ff8e.

Is this how Windows does it, as (undoc) API? (But, then, the enum definition should probably live in some .h, instead.)
https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/ntifs/nf-ntifs-cccaniwrite
says 'UCHAR / FALSE / TRUE', for what it's worth.

Or should I explicitly mark this as a local/internal extension/override only?
Or as a HACK even?

Reply:
> Is this how Windows does it, as (undoc) API? (But, then, the enum definition should probably live in some .h, instead.)

"as (undoc) API" : Yes very probably; we could probably write tests for this...

"But, then, the enum definition should probably live in some .h" : Not really if the usage is really meant to be internal as it is currently being used.